test/ReactSetStateTestComponent.js

```diff
@@ -10,68 +10,67 @@ function getNonUpdatingSetStateLifeCycleCalls() {
 
 var _nonUpdatingSetStateLifeCycleCalls = [];
 
-var ReactSetStateTestComponent = React.createClass({
-  getDefaultProps: function() {
-    return {shouldUpdate: true};
-  },
-
-  getInitialState: function() {
-    return {counter: 0};
-  },
+class ReactSetStateTestComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {counter: 1};
```

Comment:
Why was this changed from `0` to `1`?

Reply:
I never finished this portion will do that now. :P